sheets/tasks.py

```diff
@@ -0,0 +1,12 @@
+from mitol.google_sheets_refunds.api import RefundRequestHandler
+
+from main.celery import app
+
+
+@app.task
+def process_refund_requests():
+    """
+    Task to process refund requests from Google sheets
+    """
+    refund_request_handler = RefundRequestHandler()
```

Comment:
I think we should only run this task if the required settings are set and log a warning when we return